### DISCUSSION TITLE
ERRO ITMS-90035 - Air SDK 821

### DISCUSSION THREAD

**Discussion by jebarbieri** [BowlerHatLLC/vscode-as3mxml#605] [General]
Hello, I have compiled an app with version Version 33.1.1.821 using ActionScript & MXML v1.12.1 extension Bowler Hat LLC.But I can't upload it to the apple store. The same error always appears. "ERROR ITMS-90035 Invalid Signature". I have already renewed all certificates. I've searched several forums and haven't found the solution. Can you help me?

The compiled ipa is in the link below:
https://drive.google.com/file/d/1GAtj2asKRQ0tkgnWT0LcenHFBoKEdKpE/view?usp=sharing

<img width="1440" alt="itms" src="https://user-images.githubusercontent.com/38330359/167511831-56f951f2-e47d-4c52-8edc-9434b1541971.png">

**Comment by joshtynjala** [comment]
Linking to a thread in the Adobe AIR support forum from the same author. That's the more appropriate place to get help with the Adobe AIR SDK. Closing this thread in favor of that one.

https://github.com/airsdk/Adobe-Runtime-Support/issues/1898
